_posts/addons/2000-01-01-docker-image-addon.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Private Beta - Scalingo Docker Image Addon
+modified_at: 2016-05-30 00:00:00
+category: addons
+tags: docker images download feature
+---
+
+## Introduction
+
+Thanks to this addon, you'll be able to get back the docker image we're
+building when delpoying your application in our infrastructure. You
```

Comment:
Make a new point: "Use Cases". There will be may more use cases. Main one: use Scalingo as a Docker Integration Platform with staging environments hosed on Scalingo and production environment hosted elsewhere (detail elsewhere later)